go.mod

```diff
@@ -55,11 +55,12 @@ require (
 	k8s.io/code-generator v0.18.8
 	k8s.io/gengo v0.0.0-20200205140755-e0e292d8aa12
 	k8s.io/klog v1.0.0
-	knative.dev/test-infra v0.0.0-20201009204121-322fb08edae7
+	knative.dev/test-infra v0.0.0-20201013072830-525526118518
 	sigs.k8s.io/boskos v0.0.0-20200729174948-794df80db9c9
 )
 
 replace (
+	github.com/google/go-containerregistry => github.com/google/go-containerregistry v0.1.3
```

Comment:
Is this override necessary?

Reply:
look here https://github.com/knative/test-infra/issues/2497

Reply:
https://github.com/knative/pkg/pull/1802/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63R494

Reply:
With this replace this dep is excluded and we don't need to loop again in test-infra. In one or two prs, when the dep is completely excluded from the cycle, we can remove this replace

Reply:
From `eventing-kafka-broker`:

```
go get: knative.dev/test-infra@v0.0.0-20201013072830-525526118518 requires
	k8s.io/test-infra@v0.0.0-20200803112140-d8aa4e063646 requires
	knative.dev/pkg@v0.0.0-20200711004937-22502028e31a requires
	knative.dev/test-infra@v0.0.0-20200707183444-aed09e56ddc7 requires
	github.com/google/go-containerregistry@v0.1.1 requires
	github.com/goreleaser/goreleaser@v0.136.0 requires
	code.gitea.io/sdk/gitea@v0.12.0: reading code.gitea.io/sdk/gitea/gitea/go.mod at revision gitea/v0.12.0: unknown revision gitea/v0.12.0

```

Reply:
Right, I get the bump in test-infra, but I don't quite understand why we'd add a `replace` in here. It seems completely unnecessary.

Reply:
Ok let me try to add the replace in test-infra and let's see if i can get rid of that bad dep

Reply:
Check this out https://github.com/knative/test-infra/pull/2502